packages/@aws-cdk/aws-stepfunctions-tasks/lib/sns/publish.ts

```diff
@@ -92,14 +159,17 @@ export class SnsPublish extends sfn.TaskStateBase {
    * @internal
    */
   protected _renderTask(): any {
+
     return {
       Resource: integrationResourceArn('sns', 'publish', this.integrationPattern),
       Parameters: sfn.FieldUtils.renderObject({
         TopicArn: this.props.topic.topicArn,
         Message: this.props.message.value,
         MessageStructure: this.props.messagePerSubscriptionType ? 'json' : undefined,
+        MessageAttributes: this.messageAttributes?.value,
```

Comment:
can messageAttributes be expressed dynamically through a path?

i.e. is the following allowed?

```js
MessageAttributes.$: '$.somePath'
```

Reply:
I'm fine with not supporting this case